husarion_ugv_bringup/launch/bringup.launch.py

```diff
@@ -139,6 +186,37 @@ def generate_launch_description():
         }.items(),
     )
 
+    hw_config_correct = EnvironmentVariable(name="ROBOT_HW_CONFIG_CORRECT", default_value="false")
+
+    prevent_exit_action = ExecuteProcess(
+        cmd=["sleep", "infinity"],
+        condition=UnlessCondition(exit_on_wrong_hw),
+    )
+
+    incorrect_hw_config_action = GroupAction(
+        actions=[
+            error_msg(ErrorMessages.INCORRECT_HW_CONFIG),
+            prevent_exit_action,
+        ],
+        condition=UnlessCondition(hw_config_correct),
+    )
+
+    os_version = os.environ.get("SYSTEM_BUILD_VERSION", "v0.0.0")
+    os_version_correct = PythonExpression(
+        f"{check_os_version_compatibility(os_version, MIN_REQUIRED_OS_VERSION)}"
+    )
```

Comment:
PythonExpression probably can be omitted, because `os.environ` is just a string
```suggestion
    os_version_correct = check_os_version_compatibility(os_version, MIN_REQUIRED_OS_VERSION)
```

Reply:
UnlessCondition doesn't accept bool, that's why this is wrapped with PythonExpression